.travis.yml

```diff
@@ -1,7 +1,6 @@
 dist: xenial
 language: python
 python:
-  - "3.5"
```

Comment:
I would be helpful to mention in the body of the commit message that Python 3.5 is now out of support.

Could add in 3.8 as well, but that's not important for this PR.

Reply:
Thanks for the tip about adding removal of 3.5 support. I have added that note both to the CHANGES.txt file and to the commit message there.